bridge/github/export.go

```diff
@@ -117,20 +117,28 @@ func (ge *githubExporter) ExportAll(repo *cache.RepoCache, since time.Time) (<-c
 				return
 			}
 
-			snapshot := b.Snapshot()
+			select {
```

Comment:
I'm not sure you really need that, but okay

Reply:
`exportBug` handle context cancellation only it's making http requests. If all the operations are already exported the importer will not stop iterating and never catch the context cancel